components/overflow-group/overflow-group.js

```diff
@@ -0,0 +1,495 @@
+import '../colors/colors.js';
+import '../dropdown/dropdown-menu.js';
+import '../button/button.js';
+import '../button/button-subtle.js';
+import '../dropdown/dropdown.js';
+import '../dropdown/dropdown-button.js';
+import '../dropdown/dropdown-button-subtle.js';
+import '../dropdown/dropdown-more.js';
+import '../menu/menu.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { LocalizeCoreElement } from '../../lang/localize-core-element.js';
+import { offscreenStyles } from '../offscreen/offscreen.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+import { throttle } from 'lodash-es';
+
+const OPENER_TYPE = {
+	DEFAULT: 'default',
+	ICON: 'icon'
+};
+
+const OPENER_STYLE = {
+	DEFAULT: 'default',
+	SUBTLE: 'subtle',
+};
+
+/**
+ *
+ * A button group component that can be used to display a set of buttons
+ *
+ * @slot - Buttons, dropdown buttons, links or other items to be added to the container
+ */
+class OverflowGroup extends RtlMixin(LocalizeCoreElement(LitElement)) {
+
+	static get properties() {
+		return {
+			/**
+			 * Use predefined classes on slot elements to set min and max buttons to show
+			 */
+			autoShow: {
```

Comment:
Yeah when I looked into it I saw that it was using it to style based on primary buttons. I didn't know whether or not we should leave that functionality but I thought it could be useful. These functions just set new values for `minToShow` and `maxToShow` so removing it is rather easy.